pontoon/base/models.py

```diff
@@ -1370,9 +1382,13 @@ def __str__(self):
 class ProjectLocaleQuerySet(models.QuerySet):
     def available(self):
         """
-        Available project locales belong to available projects.
+        Available project locales belong to translable projects.
         """
-        return self.filter(project__disabled=False, project__resources__isnull=False).distinct()
+        return self.filter(
+            project__disabled=False,
+            project__system_project=False,
+            project__resources__isnull=False,
+        ).distinct()
```

Comment:
This change implies that we no longer need the old implementation of the `ProjectLocale.available()` method. Are you sure about that?

If yes, you should rename the method to the same name as you use on the `Project` model (e.g. `visible`).

If not, you should add a new method, instead of overriding the excisting one,